modules/docs/arrow-docs/docs/_data/sidebar-core.yml

```diff
@@ -241,6 +241,18 @@ options:
       - title: Unalign
         url: /docs/arrow/typeclasses/unalign/
 
+      - title: Zip
```

Comment:
Are these changes expected?

Reply:
Ahm, seems like they are :+1:

Reply:
Yes, those docs were the last ones included in Arrow. They must be placed in the sidebar-core section